packages/s/sqlite3/xmake.lua

```diff
@@ -48,7 +55,26 @@ package("sqlite3")
                 set_kind("$(kind)")
                 add_files("sqlite3.c")
                 add_headerfiles("sqlite3.h", "sqlite3ext.h")
-                add_defines("SQLITE_ENABLE_EXPLAIN_COMMENTS", "SQLITE_ENABLE_DBPAGE_VTAB", "SQLITE_ENABLE_STMTVTAB", "SQLITE_ENABLE_DBSTAT_VTAB", "SQLITE_ENABLE_MATH_FUNCTIONS")
+
+                if is_config("explain_comments") then
```

Comment:
options need be defined in xmake.lua

Reply:
and we need pass configs to xmake.install/configs

Reply:
ok